ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastImportBlocksStepTest.java

```diff
@@ -60,7 +60,7 @@ public class FastImportBlocksStepTest {
 
   @BeforeEach
   public void setUp() {
-    when(protocolSchedule.getByBlockNumber(anyLong())).thenReturn(protocolSpec);
+    when(protocolSchedule.getByBlockHeader(any(BlockHeader.class))).thenReturn(protocolSpec);
```

Comment:
there's a couple of other spots where anyLong() was not replaced with this class - are we losing anything by just having any()

Reply:
Changed to use any()